runtime/src/lib.rs

```diff
@@ -21,7 +21,8 @@
 // Make the WASM binary available.
 #[cfg(feature = "std")]
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
-
+extern crate alloc;
+use alloc::string::String;
 use codec::{Decode, Encode};
```

Comment:
nit: lets space these out, as they represent different domains of imports and compile time flags
```suggestion
include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));

extern crate alloc;

use alloc::string::String;
use codec::{Decode, Encode};
```